components/knative-installer/install.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+######### Knative build & serving ########
+
+echo "Installing Knative build and serving ..."
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
+
+curl -L ${SERVING_URL} \
```

Comment:
I don't like the idea we download the serving sources during installation. Can't we put it in the docker image? This way we can have quicker and repeatable installations.

Reply:
I think it's reasonable as we depend on knative release structure in scripts anyway. There is no point in having it configurable.

Reply:
Fixed